slash/plugins/builtin/xunit.py

```diff
@@ -48,10 +48,18 @@ def error_added(self, result, error):
 
     def _add_error(self, errortype):
         exc_type, exc_value, exc_tb = exc_info = sys.exc_info()
+        self._add_test_metadata(errortype,
+                                dict(type=exc_type.__name__,
+                                     message=str(exc_value)),
+                                get_traceback_string(exc_info))
+
+    def _add_test_metadata(self, title, attributes=None, content=None):
+        if not attributes:
+            attributes = {}
```

Comment:
To prevent reduced coverage, and also to test this feature, a test is needed. If you are not comfortable enough writing one I'll add it later.